### PR TITLE
🎉  catchall tidyverse docker for canine

### PR DESCRIPTION
**Pull request by dmiller15**
🔴  update circleci config

## Description

🏴‍☠️ 🐳 🐺 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Builds locally

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: tidyverse/3.6.1-canine-util/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM rocker/tidyverse:3.6.1
+LABEL maintainer="Daniel Miller (millerd15@chop.edu)"
+
+RUN install2.r optparse data.table
```
Comment (migbro) [inline tidyverse/3.6.1-canine-util/Dockerfile:4]: Fascinating

**Review by migbro — APPROVED** [review]
Fascinating

**Review by sakshamphul — APPROVED** [review]

**Review by sakshamphul — APPROVED** [review]
Note: This branch has conflicts that must be resolved
Use the [web editor](https://github.com/d3b-center/bixtools/pull/149/conflicts) or the  to resolve conflicts.
Conflicting files
.circleci/config.yml

**Comment by dmiller15** [comment]
Cool. I'll resolve those conflicts.

**Pull request merged**